website/translated_docs/de/version-14.0/networking/controlling-fixtures-over-a-network.md

```diff
@@ -0,0 +1,137 @@
+
+Einstellen der IP-Adresse des Pultes
+------------------------------------
+
+Es wird empfohlen, die integrierte Adressvergabe des Pultes zu
+verwenden. Alternativ kann aber auch manuell eine Adresse vergeben werden. 
+Siehe [Einstellen der IP-Adresse](a-quick-guide-to-ip-addressing.md#einstellen-der-ip-adresse) mit 
+Details zur IP-Adressierung.
+
+1. 	Öffnen Sie das System-Menü (mittels \<Avo\> + \<Disk\>)
+	und drücken Sie	\[Network Settings\].
+
+2.	Drücken Sie \[Local Area Connection\]. Manche Pult verfügen über mehrere
+	Netzwerkanschlüsse - wählen Sie also hier den betreffenden aus.
+
+3.	Überprüfen Sie, dass \[Subnet Mask\] auf `255.255.255.0` steht.
+
+4.	Drücken Sie \[Set IP `2.*.*.*`\].
+
+5.	Betätigen Sie \[Save settings\].
+
+6.	Verlassen Sie das System-Menü mit \<Exit\>.
+
+- 	Einige Geräte haben möglicherweise feste IP-Adressen aus dem 
+	Bereich `2.*.*.*` oder `10.*.*.*` - in diesem Fall muss auch die Adresse 
+	des Pultes in diesem Bereich liegen.
+
+-   Benötigen Sie eine Adresse aus einem anderen Adressbereich, so drücken
+	Sie auf \[IP Address = ... \] und geben die Adresse mit den Zifferntasten ein.
+
+Einrichten der DMX-Ausgänge
+---------------------------
+
+Das Pult arbeitet intern mit max. 64 DMX-Universen (16 direkt im Pult,
+bis zu 64 mit TitanNet), die einzeln auf die DMX-Buchsen sowie über
+Netzwerk-Knoten geroutet werden können. Der T1 ist auf ein, der T2 auf
+zwei Universen beschränkt. Die einzelnen Linien können sowohl über die 
+DMX-Anschlüsse als auch [über Netzwerk](../networking/controlling-fixtures-over-a-network.md) ausgegeben werden.
+
+Die Zuordnung interner DMX-Linien zu Netzwerklinien erfolgt im Menü 
+[DMX Settings](../system-settings/dmx-output-mapping.md#configuring-dmx-outputs).
+Oft wird man einfach Linie 1 auf Netzwerk-Universum 1, Linie 2 auf Universum 2 etc.
+zuweisen, aber mitunter - etwa auf Tour zur Anpassung an das vorhandene
+Netzwerk - sieht das auch ganz anders aus.
+
+Beispiel für ein einfaches Art-Net-System
+-----------------------------------------
+
+Es gibt hauptsächlich zwei Systeme zur Übertragung von Licht-Steuersignalen
+über Netzwerk: Art-Net und sACN. Titan unterstützt beide Protokolle. 
+In diesem Abschnitt wird die Einrichtung eines Art-Net-Systems erläutert.
+
+Art-Net ist kein gerätespezifisches Protokoll, sondern wird von einer
+wachsenden Anzahl von Herstellern anerkannt und in immer mehr Produkte
+integriert. Viele Geräte (etwa Dimmer oder Bewegungsscheinwerfer) können
+direkt per Art-Net verbunden werden, so dass kein separater Konverter
+erforderlich ist. Benötigt man hingegen 'normales' DMX, so kann dies
+über spezielle Art-Net-DMX-Konverter (oft als **Node** bezeichnet) 
+realisiert werden.
+
+Im nachstehenden Schema ist exemplarisch ein Art-Net-fähiges Pult (wie
+etwa ein Titan-Pult) über einen Netzwerk-Switch mit einem Art-Net-fähigen
+Dimmer und einem Konverter verbunden.
+
+![Art Net Explanation
+general](/docs/images/Art-Net-Explanation-General.jpeg)
```

Comment:
I think it's unhappy about this being a capital G. Should be changed to `/docs/images/Art-Net-Explanation-general.jpeg`

Reply:
Or the file renamed... but yes it is case sensitive